### DISCUSSION TITLE
Developer role should be available for Pro plan

### DISCUSSION THREAD

**Discussion by islishude** [vercel/community#3834] [Idea]
### Question

I can set owner role for our dev team.

and the permission of owner role is excessive.

I need developer role for my Pro plan!

**Comment by amyegan** [comment]
Hey @islishude. Thanks for the feedback. Pro plans currently have access to the following roles:
- [Owner Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#owner-role) - full access
- [Member Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#member-role) - cannot edit team settings, invite new users to team, change roles of other users on team, or manage billing info
- [Billing Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#billing-role) - can _only_ manage billing info

The Enterprise [Developer Role](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#developer-role) is similar to Member except that it also restricts their ability to manage project settings and deploy to production. If you don't need these additional restrictions, then the Member Role may be a good fit.

**Reply by islishude** [reply]
But I need it.

I don't want everyone to be able to manage project and lookup environment variables on production.

so developer role is suitable for me.

your enterprise plan should have a fine-grained roles rather than fixed role your provided.

Free developer role for pro plan.
